libraries/listen.rb

```diff
@@ -34,8 +34,8 @@ def self.converted_listen_ports_and_addresses(node)
       return [] unless node['apache']['listen_ports'] || node['apache']['listen_addresses']
```

Comment:
I did not realize that they changed it to `||` in 3.2.2 (changeset 52372bbe74c25acea1021eb5f7d1d1df449c4fd6). In that case it probably needs some sane default for ports and addresses if either are nil. Maybe what it had in 3.1?